Gemfile.lock

```diff
@@ -732,9 +732,3 @@ DEPENDENCIES
   whenever
   wicked_pdf
   wkhtmltopdf-binary
-
-RUBY VERSION
-   ruby 2.1.5p273
-
-BUNDLED WITH
-   1.12.5
```

Comment:
Was there a reason for modifying `Gemfile.lock`? I think it's one of those files that doesn't usually get touched.